src/stylesheets/app.scss

```diff
@@ -10,7 +10,6 @@ $fontfamily: 'Open Sans', sans-serif;
 @import "colors";
 @import "global";
 
-@import "admin_auth_service_edit_form";
```

Comment:
It doesn't look like the corresponding `admin_auth_service_edit_form.scss` file has been removed. Do we need to keep that around for anything else?

Reply:
Good catch 👍 https://github.com/ThePalaceProject/circulation-admin/pull/75/commits/57649772776b7fea9d2c9358550fe76f90312706